src/com/tutorial/main/ID.java

```diff
@@ -1,9 +0,0 @@
-package com.tutorial.main;
-
-public enum ID {
-
-	Player(),
-	BasicEnemy(),
-	Trail();
```

Comment:
Enum values are generally defined with all caps (as with all other constants). The parentheses are unneeded in this case since you don't have a parameterized constructor for this enum.